tests/ui/result_map_or_into_option.rs

```diff
@@ -1,8 +1,13 @@
-#![warn(clippy::result_map_or_into_option)]
+#![deny(clippy::result_map_or_into_option)]
```

Comment:
IIRC, we usually only put `warn` attributes here, since `ui_test` will supply the `-Dwarnings` anyway.

Reply:
Reverted this line.